src/pymor/reductors/interpolation.py

```diff
@@ -68,16 +68,32 @@ def __init__(self, fom, mu=None):
         self._product = None
 
     def _B_apply(self, s, V):
-        return self.fom.transfer_function.B(s).apply(V, mu=self.mu)
+        mu = Mu({
+            's': [s],
+            **self.mu
+        })
```

Comment:
Using the `|` operator looks a bit nicer (similar below).

```suggestion
        mu = Mu({'s': [s]} | self.mu)
```

Reply:
Actually, `|` was introduced in Python 3.9, so we shouldn't use it, at least for now. The four lines could still fit comfortably in one line, I think.